ghmirror/core/mirror_requests.py

```diff
@@ -58,6 +58,64 @@ def _cache_response(resp, cache, cache_key):
         cache[cache_key] = resp
 
 
+def _online_request(method, url, data=None,
```

Comment:
This is a wrapper for `GET` requests to mark them as valid or invalid. I would name it accordingly